test/evergreen/download_metrixpp.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# This script clones Metrix++ and checks out a specific commit
+git clone https://github.com/metrixplusplus/metrixplusplus metrixplusplus || exit
+
+cd metrixplusplus || exit
+
+# Check out the version from 18th Dec 2022, which is the latest commit as of 28th Feb 2024.
+git checkout 78dc5380de9aaa3d615f8be6c84e90cb2ae0d90b || exit
```

Comment:
Any reason to checkout a particular commit ? I believe earlier we were just checking out the latest metrixplusplus.

Reply:
Hi Ravi, I'm keen that we switch to a specific version of Metric++ so that we can control upgrades (including if they have any issues). Metrix++ doesn't have version numbers or releases, so a git hash is the next best thing. 
I'll expand the comment to make this clearer for future readers.